WcaOnRails/app/controllers/regulations_controller.rb

```diff
@@ -3,9 +3,26 @@
 class RegulationsController < ApplicationController
   include HighVoltage::StaticPage
 
+  before_action :ensure_trailing_slash
+
   private def page_finder_factory
     IndexPageFinder
   end
+
+  private def ensure_trailing_slash
+    desired_url = url_for(params.permit!.merge(trailing_slash: true))
```

Comment:
Yep! For the record, the request spec I added in this PR failed before I added this call to `permit!`